### ISSUE TITLE
🐛 Disks won't eject 

### ISSUE THREAD

**Issue by jmkuehn** [automatic-ripping-machine/automatic-ripping-machine#1051]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### I have checked the wiki

- [X] I have searched the wiki for this problem

### Does this issue exist in the latest version?

- [X] I'm using the latest release

### Describe the bug?

Drives will not automatically or manually eject disks. 

### To Reproduce

I'm using TrueNAS SCALE with the image from TrueCharts catalog.
I have /dev/sr0 mounted to /dev/sr0
Everything works fine 
going to Arm Settings > General Info > Disk drives and press Eject - nothing happens 

### What OS are you seeing the problem on?

Other (please state in Anything else? section)

### ARM Release Version or Git hash (You can find this in the arm settings page)

v2.6.70

### Expected behavior?

Drive to eject

### Relevant log output

```shell
2024-01-31 18:06:46.428687-05:00eject: tried to use udev' as device name but it is no block device
2024-01-31 18:06:46.428913-05:00eject: tried to use /mnt//dev/sr1' as device name but it is no block device
2024-01-31 18:06:46.429180-05:00eject: unable to find or open device for: /dev/sr1'
2024-01-31 18:06:46.429235-05:00eject: device name is /dev/sr1'
2024-01-31 18:06:46.429263-05:00eject: expanded name is /dev/sr1'
2024-01-31 18:06:46.429271-05:00eject: udev' is mounted at /dev/sr1'
2024-01-31 18:06:46.429278-05:00eject: expanded name is /mnt//dev/sr1'
2024-01-31 18:06:46.429285-05:00eject: /mnt//dev/sr1' is not mounted
2024-01-31 18:06:46.429291-05:00eject: /mnt//dev/sr1' is not a mount point
```

### Anything else?

![image](https://github.com/automatic-ripping-machine/automatic-ripping-machine/assets/8756916/ff089e36-b29a-43d7-a535-1a28687a0070)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by microtechno9000** [comment]
I have seen a few issues with TrueNAS of recent. Unfortunately the Devs don't run TrueNAS and don't have systems to test ARM.

Whilst the move to docker has helped resolve compatibility issues, it seems TrueNAS has issues with the ARM config. If you or any other ARM users have a system and programming knowledge we welcome any feedback when running TrueNAS.

To help with your issue. are you able to run 'eject /dev/sr0' ? from the host system or within docker?

**Comment by jmkuehn** [comment]
Host system: 
Linux old-fashioned 6.1.63-production+truenas #2 SMP PREEMPT_DYNAMIC Tue Jan 23 22:31:02 UTC 2024 x86_64

        TrueNAS (c) 2009-2024, iXsystems, Inc.
        All rights reserved.
        TrueNAS code is released under the modified BSD license with some
        files copyrighted by (c) iXsystems, Inc.

        For more information, documentation, help or support, go here:
        http://truenas.com

Welcome to TrueNAS
Last login: Wed Jan 31 18:03:18 EST 2024 on pts/13

Warning: the supported mechanisms for making configuration changes
are the TrueNAS WebUI and API exclusively. ALL OTHERS ARE
NOT SUPPORTED AND WILL RESULT IN UNDEFINED BEHAVIOR AND MAY
RESULT IN SYSTEM FAILURE.

root@old-fashioned[~]# eject /dev/sr1
zsh: command not found: eject
root@old-fashioned[~]# 


opening the shell for the app I get:

WARNING: Your user does not have sudo privileges so /usr/local/bin/k3s command will run
on your behalf. This might cause permission issues.

# eject /dev/sr1
eject: tried to use `udev' as device name but it is no block device
eject: tried to use `/mnt//dev/sr1' as device name but it is no block device
eject: unable to find or open device for: `/dev/sr1'
# 

But eject /mnt/dev/sr1 will work as expected with no output

**Comment by jmkuehn** [comment]
This issue appears to have been resolved by truecharts release 2.1.30.

**Comment by jmkuehn** [comment]
It worked once, after a disk errored out, but the previous bad behavior has continued
